### ISSUE TITLE
onLeftClickEntity - Items

### ISSUE THREAD

**Issue by clienthax** [MyEssentials/MyTown2#499]
Seems the support for this function is missing

**Comment by AfterWind** [comment]
So it's an `ITEM` that you need to protect from being `LEFT CLICKED` on an `ENTITY`, correct?
If so, then yeah, that's missing.

**Comment by clienthax** [comment]
Yes. its a function in the Item class, added by forge by looks of it
However i don't see a event in forge for it =/
